TotemRouteServiceProvider.php with custom uri

frnwtr opened the pull request:
enable user to customize the totem uri by settings

frnwtr committed:
TotemRouteServiceProvider.php with custom uri

enable user to customize the totem uri by settings

qschmick commented on src/Providers/TotemRouteServiceProvider.php:
```diff
@@ -66,9 +69,12 @@ protected function mapWebRoutes()
-            ->namespace($this->namespace)
-            ->group(__DIR__.'/../../routes/api.php');
+        Route::group([
+            'prefix' => config('totem.uri', 'totem').'/api',
```
Update this to:
`'prefix' => config('totem.uri', 'totem').'/'.config('totem.api_prefix', 'api'),`

frnwtr replied:
seems legit

qschmick committed:
Update TotemRouteServiceProvider.php

Updated configurable uri to also allow the api prefix to be fully configurable.

roshangautam commented:
Closing due to lack of activity
